interfaces/Portalicious/src/app/pages/project-payments/components/create-payment/create-payment.component.ts

```diff
@@ -282,10 +295,9 @@ export class CreatePaymentComponent {
       label: $localize`Export payment list`,
       icon: 'pi pi-upload',
       command: () => {
-        // TODO: AB#31502
-        this.toastService.showToast({
-          severity: 'warn',
-          detail: "Haven't implemented this yet 😵‍💫",
+        this.exportRegistrationsMutation.mutate({
+          type: ExportType.allPeopleAffected,
```

Comment:
Are we sure this is the correct type? I would've expected `ExportType.payment` here

Reply:
Yup - this action is the equivalent of exporting from the registrations table. AFAIU we can't use `ExportType.payment` because the payment hasn't been created in the DB yet.